### ISSUE TITLE
where can we download the weights of MIMR reward?

### ISSUE THREAD

**Issue by JoshonSmith** [bytedance/UMO#7]
1. where can we download the weights of MIMR reward?
```
# detect fact to get bbox
self.det_net = init_detection_model('retinaface_resnet50', half=False, device=device, model_rootpath='models/pulid/facexlib').requires_grad_(False).to(device, dtype=weight_dtype)
# get face emb
self.arcface_model = get_arcface_model(model_name='r50', pretrained_path='models/pulid/glint_cosface_res50.pth').to(device, dtype=weight_dtype)
```

2. How can we use MIMR class if we have two or more reference human images?
```
def score_grad(self, image: torch.Tensor, image_gt: torch.Tensor, **kwargs):
```

**Comment by cb1cyf** [comment]
1. `det_net` would be downloaded automatically with function `init_detection_model`; you can download `arcface_model` from https://github.com/deepinsight/insightface/tree/master/recognition/arcface_torch

<img width="394" height="464" alt="Image" src="https://github.com/user-attachments/assets/d70da4b6-6ad5-4348-9459-2b82cbe8763a" />

2. `image` is the tensor of generated image, `image_gt` is the tensor of ground truth image which contains all reference identities. If you want to use MIMR without the ground truth image, `i.e.`, compute MIMR between generated image and several reference images, a simple way is getting face embeddings for each reference image, then concat or stack them to get `face_emb_gt` in https://github.com/bytedance/UMO/blob/main/reward_models/MIMR/multi_id_matching_reward.py#L88
